node-operators/block-producers/operational-tasks/upgrade-your-node/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Upgrade Block Producers
-description: Follow these instructions to update your node to the latest version of the Tanssi client software you use to produce blocks on the Tanssi Appchain protocol.
+description: Follow these instructions to update your node to the latest version of the Tanssi client software you use to produce blocks on the Tanssi Network protocol.
```

Comment:
```suggestion
description: Follow these instructions to update your node to the latest version of the Tanssi client software you use to produce blocks on the Tanssi network protocol.
```

Reply:
there are some inconsistencies with Network capitalized. search for it and check it is OK